apis/record_auth.go

```diff
@@ -160,7 +161,7 @@ func (api *recordAuthApi) authMethods(c echo.Context) error {
 		}
 
 		state := security.RandomString(30)
-		codeVerifier := security.RandomString(43)
+		codeVerifier := base64.URLEncoding.EncodeToString([]byte(security.RandomString(43)))
```

Comment:
There is no need to base64 encode the codeVerifier. I've tested it it works fine with just the random string.

Reply:
okay, then I guess I must have done something weird client side! I'll give it another shot